flaskbb/user/plugins.py

```diff
@@ -0,0 +1,22 @@
+from itertools import chain
+from pluggy import HookimplMarker
+
+impl = HookimplMarker('flaskbb')
+
+
+@impl(hookwrapper=True, tryfirst=True)
```

Comment:
Marked tryfirst in case other folks don't heed the warning in the spec description about not implementing their own hookwrappers around this.